flake.nix

```diff
@@ -5,6 +5,9 @@
     nixpkgs.url = "github:nixos/nixpkgs?rev=b1f87ca164a9684404c8829b851c3586c4d9f089";
     unstable.url = "github:nixos/nixpkgs/nixos-unstable";
     deploy-rs.url = "github:serokell/deploy-rs";
+    vscode-server.url = "github:nix-community/nixos-vscode-server";
+    boba-backend.url = "github:bobaboard/boba-backend/devenv";
+    boba-frontend.url = "github:jakehamilton/boba-frontend/hack/nix";
```

Comment:
All of these inputs (except nixpkgs and unstable) should have `follows` set. Otherwise they'll pull in their own versions of NixPkgs.

Reply:
Annoying, but done :)

Reply:
Mood :(